.travis.yml

```diff
@@ -3,21 +3,5 @@ dist: trusty
 language: python
 python:
   - 3.5
-install:
-  - sudo apt-get update
-  - wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
-  - bash miniconda.sh -b -p $HOME/miniconda
-  - export PATH="$HOME/miniconda/bin:$PATH"
-  - hash -r
-  - conda config --set always_yes yes --set changeps1 no
-  - conda update -q conda
-  # Useful for debugging any issues with conda
-  - conda info -a
-
-  # Replace dep1 dep2 ... with your dependencies
-  - conda create -q -n test-environment python=3.5 numpy scipy pandas nose nbformat nbconvert jupyter_client jupyter matplotlib
-  - source activate test-environment
-  - pip install https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-1.2.1-cp35-cp35m-linux_x86_64.whl
-  - python setup.py install
 script:
-  - nosetests -v --nologcapture testing
+  - echo "Hello world!"
```

Comment:
Is this just here to disable travis for the time being?

Reply:
yep, I didn't want to run travis agains raw branch